### ISSUE TITLE
Update or remove links to UK Notify API documentation

### ISSUE THREAD

**Issue by ccostino** [GSA/notifications-admin#564]
The API documentation page currently contains a handful of links that point to the UK Notify's API documentation:

<img width="1096" alt="Screen Shot 2023-06-20 at 11 39 54 AM" src="https://github.com/GSA/notifications-admin/assets/121565/640b82ce-dc91-4102-8c38-905568465df0">

If we're not leveraging them as is, we'll want to update or remove them in the future.

**Comment by tdlowden** [comment]
I *think* the only place you can find this is in the footer, correct @ccostino?

Previous content changes removed this page from view. I believe we either need to deactivate the page altogether or at minimum just remove the link/content from the footer.

**Comment by ccostino** [comment]
Yes, I got to it from the footer, @tdlowden!  Just removing the link is a quick fix; removing the view associated with the link a bit more work but more complete so you can't get to it at all.

@maomeara63 also mentioned in stand-up today that this may already be accounted for in another issue - if that's the case, we can close this one!
